### PR TITLE
Install fonts and define typeface constants

### PR DESCRIPTION
**Pull request by Honglin-Lu**
1. Install two fonts (Roboto and Merriweather) through `npm install` and import them in `preview.ts` file and also put them in the peerDependency.
2. Design system tokens for typography goes under the typography field.

### CODE REVIEW

File: src/theme/typography.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  interface TypographyPropsVariantOverrides {
+    subheader1: true;
+    subtitle1: false;
+    subheader2: true;
+    subtitle2: false;
+    subheader3: true;
+    subheader4: true;
+  }
+}
+
```
Comment (wenfeng-xing) [inline src/theme/typography.ts:31]: I am not sure if we need to create a theme only for typography, maybe we need to talk about it.

File: src/theme/typography.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+export const typography = createTheme({
```
Comment (wenfeng-xing) [inline src/theme/typography.ts:32]: I am not sure if we need to create a theme only for typography, maybe we need to talk about it.

**Review by tprince232 — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -34,9 +34,13 @@
...
     "@mui/material": "^5.10.6",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "@fontsource/merriweather": "^4.5.14",
+    "@fontsource/roboto": "^4.5.8"
   },
   "devDependencies": {
+    "@fontsource/merriweather": "^4.5.14",
+    "@fontsource/roboto": "^4.5.8",
```
[the remark this answers is not in the document]
Comment (Honglin-Lu) [reply package.json]: Solved through Slack communication

File: .storybook/preview.ts [hunk 4]
```diff
@@ -1,4 +1,8 @@
 import { themes } from '@storybook/theming';
+import '@fontsource/roboto/300.css';
+import '@fontsource/roboto/400.css';
+import '@fontsource/roboto/500.css';
+import '@fontsource/merriweather/400.css';
```
[the remark this answers is not in the document]
Comment (Honglin-Lu) [reply .storybook/preview.ts]: Solved through Slack communication.

**Review by tprince232 — APPROVED** [review]

**Pull request merged**